src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/WebSocketsTransport.cs

```diff
@@ -66,69 +67,51 @@ public WebSocketsTransport(WebSocketOptions options, IChannelConnection<byte[]>
                 receiving,
                 sending);
 
-            // What happened?
+            var failed = false;
+            var task = Task.CompletedTask;
             if (trigger == receiving)
             {
                 if (receiving.IsCanceled || receiving.IsFaulted)
```

Comment:
I added it back because if the client disconnects abnormally via `socket.Abort()` like one of our tests does, then `sending` never completes and we just timeout on the server. If that's the behavior we want I can change the test to not expect an `OperationCanceledException`